tools/project_manager/remote_file_manager.php

```diff
@@ -737,14 +737,14 @@ function is_valid_move_destination($dir)
  * removing empty components, rejecting parent traversal,
  * and re-joining them.
  *
- * Returns False is path contains invalid components.
+ * Returns null is path contains invalid components.
  */
-function canonicalize_path($relpath)
+function canonicalize_path(string $relpath): ?string
```

Comment:
This function and `get_directory_items_sorted` were switched from returning `false` on error to returning `null` as it yields to be a better type. I updated the callers, which were all in this file, to check against `null`.